### PR TITLE
Move acceptor models to the shared folder

### PR DESCRIPTION
**Pull request by mykola-pysarev**
Issue #<Number of the issue>, <SHORT DESCRIPTION>

Changes:
1. Has moved models from the acceptor to shared folder;
2. Modified messageSender to support models mandatory fields;
3. Has fixed tests to support models movements;

Notes:
- notes 1;
- notes ..;
- notes N.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ install:
         - pip install -r command-processor/requirements.txt
         - pip install -r telegram-pusher/requirements.txt
 script:
-        - python -m unittest discover command-acceptor/tests -p *tests.py -s command-acceptor
+        - PYTHONPATH=./shared python -m unittest discover command-acceptor/tests -p *tests.py -s command-acceptor
+        #- python -m unittest discover command-acceptor/tests -p *tests.py -s command-acceptor
```
Comment (andrii-z4i) [inline .travis.yml]: remove it

File: shared/model/AcceptedMessage.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from jsonobject import JsonObject, IntegerProperty, ObjectProperty, StringProperty
-from app.models.Chat import Chat
-from app.models.User import User
+from shared.model.Chat import Chat
```
Comment (andrii-z4i) [inline shared/model/AcceptedMessage.py]: why shared?

File: .travis.yml (hunk 1 above)
Comment (mykola-pysarev) [reply .travis.yml]: Done

File: shared/model/AcceptedMessage.py (hunk 2 above)
Comment (mykola-pysarev) [reply shared/model/AcceptedMessage.py]: Done

**Comment by andrii-z4i** [comment]
Please modify message commit before merge. @mykola-pysarev

**Pull request merged**